ApiDocs/fetch.apidoc

```diff
@@ -1,5 +1,5 @@
 /**
- * @api {endpoint / fetch} /io.scalecube.configuration.api.ConfigurationService/fetch Fetch (get) specified entry
+ * @api {endpoint / fetch} /io.scalecube.configuration.api.ConfigurationService/fetch Get specified entry
```

Comment:
need to change the name of the using alias name ```configuration``` and not using the class name:
```/io.scalecube.configuration.api.ConfigurationService```

seem it was left as mistake

Reply:
Ok, but then it wouldn't be aligned with `https://raw.githubusercontent.com/scalecube/scalecube-configuration-service/master/API-Calls-examples.json` as it serves as import-template for sandbox.